wgsl/TSPath.py

```diff
@@ -117,13 +117,13 @@ def __init__(self,subexpr):
     def match(self,ts_node):
         result = []
         # Stack of treesitter nodes to explore
-        stack = [ts_node]
+        stack = list(reversed(ts_node.children))
```

Comment:
The functional fix is to start with the children.
Otherwise searches like '//template_list//template_list' would not work.

The reversing is cosmetic: it causes the traversals to be left-to-right, since this is a stack.